lib/interface/cli/commands/hybrid/delete.cmd.js

```diff
@@ -35,10 +35,10 @@ const deleteCmd = new Command({
             default: DEFAULTS.URL,
         })
         .option('kube-context-name', {
-            describe: 'Name of the kubernetes context on which venona should be installed [$CF_ARG_KUBE_CONTEXT_NAME]',
+            describe: 'Name of the kubernetes context on which venona should be uninstalled [$CF_ARG_KUBE_CONTEXT_NAME]',
```

Comment:
Name of the kubernetes context from which the Codefresh Agent and Runtime should be removed